release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -515,6 +515,15 @@ Enabling the `TechPreviewNoUpgrade` feature set on your cluster cannot be undone
 
 For more information, see xref:../nodes/pods/nodes-pods-configuring.adoc#pod-disruption-eviction-policy_nodes-pods-configuring[Specifying the eviction policy for unhealthy pods].
 
+[id="ocp-4-13-graceful-node-shutdown"]
+==== Support for graceful node shutdown
+
+A graceful node shutdown delays the eviction of pods during a node shutdown. In {product-title} {product-version}, you can configure the kubelet to enable a graceful node shutdown to ensure that pods running critical workloads are not interrupted.
```

Comment:
Just thinking this might be a little clearer. Up to you!
```suggestion
A graceful node shutdown delays the eviction of pods during a node shutdown. In {product-title} {product-version}, you can configure the kubelet to enable a graceful node shutdown so pods running critical workloads are not interrupted.
```